auctionapp/auction-app
# complete basic functionality for SellItem component form - chris

#### What's this PR do?

Complete all of basic functionality for SellItem component
#### Where should the reviewer start?

client/components/SellItem.jsx, server/app.js, server/db/models/item.js
#### How should this be manually tested?

Use the form on the front page to submit information and then check the actual database to see the data pop up
#### Any background context you want to provide?

No
#### What are the relevant tickets?

n/a
